src/Transition/OutputRulesFiles/OutputChanges24-1-0-to-24-2-0.md

```diff
@@ -38,3 +38,7 @@ with
 - Maximum for Winter Design Days
 
 These changes will also make corresponding changes in the HTML Intialization Summary report
+
+### Adding an Output:Variable, Wetbulb Globe Temperature
+
+An output variable will be added, Wetbulb Globe Temperature.
```

Comment:
This should list both flavors, Zone Wetbulb Globe Temperature and Space . . .

Reply:
I'll add the space version too